Source/monster.cpp

```diff
@@ -3831,10 +3831,8 @@ void monster_some_crypt()
 void InitMonsters()
 {
 	if (!setlevel) {
-		AddMonster(GolemHoldingCell, DIR_S, 0, false);
-		AddMonster(GolemHoldingCell, DIR_S, 0, false);
-		AddMonster(GolemHoldingCell, DIR_S, 0, false);
-		AddMonster(GolemHoldingCell, DIR_S, 0, false);
+		for (int j = 0; j < MAX_PLRS; j++)

```

Comment:
Can't we use something like `for (auto _ : players)` here and avoid the extra dependency with `MAX_PLRS`?

Reply:
That seams really missleading

Reply:
I don't know. I think the `MAX_PLRS` macro should not exist.

Reply:
Not disagreeing, but that is a refactor for a later day :) Just pulled this out of another PR so that it could get closed.